plugins/junit/src/com/intellij/execution/junit/codeInsight/references/JUnitReferenceContributor.java

```diff
@@ -34,6 +34,12 @@
     return PlatformPatterns.psiElement(PsiLiteral.class).and(new FilterPattern(new TestAnnotationFilter(annotation, paramName)));
   }
 
+  private static PsiElementPattern.Capture<PsiLiteral> getEnumSourceNamesPattern() {
+    return PlatformPatterns.psiElement(PsiLiteral.class)
+      .and(new FilterPattern(new TestAnnotationFilter(JUnitCommonClassNames.ORG_JUNIT_JUPITER_PARAMS_ENUM_SOURCE, "names")))
```

Comment:
that EnumSource annotation is marked as experimental, but imo widly used and I tried to write code to do nothing when api changes (i hope :]) in the future